ksqldb-engine/src/main/java/io/confluent/ksql/internal/PullQueryExecutorMetrics.java

```diff
@@ -475,10 +477,9 @@ private void addRequestMetricsToSensor(
     );
 
     sensor.add(new Percentiles(
-        100,
-        0,
-        1000,
-        BucketSizing.CONSTANT,
+        4 * NUM_LATENCY_BUCKETS,
```

Comment:
Where does the 4 come from?

Reply:
Conversion of buckets to bytes, which is what the api asks for...  I wish they would just change the API.  I'll add a comment.